### PR TITLE
Enrichment filter dict

### PR DESCRIPTION
**Pull request by oleurud**
Solves https://github.com/CartoDB/cartoframes/issues/1374 & https://github.com/CartoDB/cartoframes/issues/1390

### CODE REVIEW

File: cartoframes/data/observatory/enrichment/enrichment.py [hunk 1]
```diff
@@ -38,8 +38,11 @@ def enrich_points(self, dataframe, variables, geom_col=None, filters=[]):
                 variable ID, slug or :obj:`Variable` instance or list of variable IDs, slugs
                 or :obj:`Variable` instances taken from the Data Observatory :obj:`Catalog`.
             geom_col (str, optional): string indicating the geometry column name in the source `DataFrame`.
-            filters (list, optional): list of :obj:`VariableFilter` to filter rows from
-                the enrichment data. Example: `[VariableFilter(variable1, "= 'a string'")]`
+            filters (dictionary, optional): dictionary to filter results by variable values. As a key it receives the
```
Comment (Jesus89) [inline cartoframes/data/observatory/enrichment/enrichment.py]: dictionary -> dict

File: cartoframes/data/observatory/enrichment/enrichment.py [hunk 2]
```diff
@@ -148,8 +151,11 @@ def enrich_polygons(self, dataframe, variables, geom_col=None, filters=[], aggre
...
```
Comment (Jesus89) [inline cartoframes/data/observatory/enrichment/enrichment.py]: dictionary -> dict

File: cartoframes/data/observatory/enrichment/enrichment.py [hunk 3]
```diff
@@ -260,12 +266,28 @@ def enrich_polygons(self, dataframe, variables, geom_col=None, filters=[], aggre
...
+
+            .. code::
+
+                from cartoframes.data.observatory import Enrichment, Variable
+
+                variable = Variable.get('no_cars_d19dfd10')
+                enriched_dataset_cdf = Enrichment().enrich_polygons(
+                    my_local_dataframe,
+                    variables=[variable],
+                    aggregation=[VariableAggregation(variable, 'SUM')]
```
Comment (Jesus89) [inline cartoframes/data/observatory/enrichment/enrichment.py]: aggregation should be using `dict` too, right?

**Comment by oleurud** [comment]
I need to merge it to continue with e2e tests. In case you find something, please ping me

**Pull request merged**